src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.header__1200 {
```

Rename this selector. It must not contain numbers in the name.
```suggestion
.header {
```